python-project-template/README.md.jinja

```diff
@@ -0,0 +1,4 @@
+# {{project_name}}
+
+This project was automatically generated using the LINCC-Frameworks python-project-template.
```

Comment:
Can we include a link to the project here?

Reply:
Sure thing.